src/pages/guidelines/ibm-dot-com-style-models/overview.mdx

```diff
@@ -0,0 +1,131 @@
+
+
+Here are the three principles that we set up to reach our vision of helping our users in the best way possible.
+
+<Title>1. The experience is the product</Title>
+
+The user&lsquo;s experience along the journey—spanning from the user&lsquo;s initial research through decision making, purchase, management, and all the way to the actual use of the given product—is becoming a core aspect of any product. 
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![User journeys](./images/journey-non-linear.png)
+
+</Column>
+</Row>
+
+<Title>2. Drive consistency around user intents</Title>
+
+As more products are offered as services or components of larger ecosystems and platforms, users are increasingly traversing journey touch points in non-linear ways, which demands a even higher level of consistency and coherency across journeys.
+
+
+We drive consistency without sacrificing usability by basing the design of a page on the given user intent. For the same user intent, whether it&lsquo;s learning, trying, or buying, the solution should be consistent in order to accommodate the user&lsquo;s non-linear journey. This does not mean that everything should be the same—IBM.com style models define foundational and structural elements, while leaving the details to you, the maker.
+
+
+The ultimate end result is that the user should be instantly familiar with pages that solve similar problems.
+
+<Title>3. Reflect users&rsquo; journeys</Title>
+
+The design should reflect the progress the user is making towards the end goal of purchasing and using a product to solve their problems and achieve objectives. The style models reflect  The design changes across toucpoints and their underlying style models should be coherent with each other, and reflect the progress the user is making.
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![Reflecting journey progression](./images/journey-progression.png)
+
+</Column>
+</Row>
+
+
+## Style model
+A style model defines the foundational structure and elements of a given page to best serve the primary user intent of that page.
+
+
+Each style model is based on the user intent for a given page and defines the most optimal use of the foundational design elements—grid, color theme, type scale, spacing, layout, and screen regions—to ensure that the user is successful in achieving their goals (user intent).
+
+<Title>Primary user intent</Title>
+
+For any given page, there is a overarching user intent—we call this the primary user intent of a page. For an example, learn is the primary user intent for the early learn search capture page.
+
+
+<Title>Grid usage</Title>
+
+The grid system creates the fundamental structure of a page. There are many different ways to use the IBM 2X Grid, and the style models narrow down the available options to a few optimized.
```

Comment:
"...to a few optimized" what? 
Layouts
grid modes(?)
optimized for editorial layouts?

Reply:
How about "a few optimized structures"?